packages/devtools_app/lib/src/profiler/cpu_profile_service.dart

```diff
@@ -27,7 +27,7 @@ class CpuProfilerService {
         .clearCpuSamples(serviceManager.isolateManager.selectedIsolate.id);
   }
 
-  Future<Success> setProfilePeriod(String value) {
+  Future<dynamic> setProfilePeriod(String value) {
```

Comment:
@bkonyi is this intentional (`Success -> dynamic`)

Reply:
Yes, see https://github.com/dart-lang/sdk/commit/35748514c724a3da9c45d8b45b580f3594eb179e